source/administration/bulk-export-data.rst

```diff
@@ -0,0 +1,322 @@
+Bulk export data
+=================
+At this time, core properties of non-deleted objects listed below are exported.  Configuration for exporting specific areas of the server, exporting additional types of posts, reactions, file attachments, webhooks and bot messages are not yet supported. 
+
+Version object
+--------------
+.. raw:: html
+
+  <table width="100%" border="1" cellpadding="5px" style="margin-bottom:20px;">
+    <tr class="row-odd">
+      <th class="head">Field name</th>
+      <th class="head">Type</th>
+      <th class="head">Description</th>
+    </tr>
+    <tr class="row-odd">
+      <td valign="middle">type</td>
+      <td valign="middle">string</td>
+      <td>Must be the string "version"</td>
+    </tr>
+    <tr class="row-odd">
+      <td valign="middle">version</td>
+      <td valign="middle">number</td>
+      <td>Must be the number 1.</td>
+    </tr>
+  </table>
+  
+Team object
+-----------
+.. raw:: html
+
+  <table width="100%" border="1" cellpadding="5px" style="margin-bottom:20px;">
+    <tfoot><tr><td colspan="5">[1] Not validated, but an error occurs if no such scheme exists when running in apply mode.</td></tr></tfoot>
```

Comment:
Is this footnote for a specific field, or for the object in general?

Reply:
Removed; missed when I copied the table formatting from another location.